astroquery/gemini/core.py

```diff
@@ -433,6 +434,97 @@ def get_file(self, filename, *, download_dir='.', timeout=None):
         local_filepath = os.path.join(download_dir, filename)
         self._download_file(url=url, local_filepath=local_filepath, timeout=timeout)
 
+    def _download_file_content(self, url, timeout=None, auth=None, method="GET", **kwargs):
```

Comment:
Make optional arguments kwarg only. Also, what are the possible keys in kwargs, explicitly list them if possible.
```suggestion
    def _download_file_content(self, url, *, timeout=None, auth=None, method="GET", **kwargs):
```

Reply:
The kwargs could be any accepted keys in  the underlying `Session.request`. I can make a note of them.